static/app/views/replays/detail/errorList/errorTableCell.tsx

```diff
@@ -131,16 +132,23 @@ const ErrorTableCell = forwardRef<HTMLDivElement, Props>(
       () => (
         <Cell {...columnProps}>
           <Text>
-            <QuickContextHoverWrapper
-              dataRow={{
-                id: eventId,
-                'project.name': projectSlug,
-              }}
-              contextType={ContextType.EVENT}
-              organization={organization}
-            >
-              {title ?? EMPTY_CELL}
-            </QuickContextHoverWrapper>
+            <ClassNames>
+              {({css}) => (
+                <QuickContextHovercard
+                  dataRow={{
+                    id: eventId,
+                    'project.name': projectSlug,
+                  }}
+                  contextType={ContextType.EVENT}
+                  organization={organization}
+                  containerClassName={css`
+                    display: inline;
+                  `}
```

Comment:
😬 Doesn't this mean that `QuickContextHovercard` will now have a worse default, as it seems to generally require `HoverWrapper`? Would it work if we were able to style `HoverWrapper` from this component instead?

Reply:
I think the benefit here is we're splitting up the pop-over part from the think that triggers the popover. To me the previous version of `QuickContextHovercard` was akin to sub-classing, where we've got all these things coming together, but if we want to customize anything then we have to pass in 


But I should keep the old `*Wrapper` file, just extract out the hovercard part. That way all the existing files don't need to be touched.